packages/api/src/controllers/stream.test.ts

```diff
@@ -1134,15 +1170,63 @@ describe("controllers/stream", () => {
         const res = await client.get(`/stream/${document.id}`);
         expect(res.status).toBe(200);
       }
+
+      // create a new project
+      client.jwtAuth = nonAdminToken;
+      projectId = await createProject(client);
+      expect(projectId).toBeDefined();
+
+      // then create a new api-key under that project
+      const allowedOrigins = ["http://localhost:3000"];
+      newApiKey = await createApiToken({
+        client: client,
+        projectId: projectId,
+        jwtAuthToken: nonAdminToken,
+      });
+      expect(newApiKey).toMatchObject({
+        id: expect.any(String),
+      });
+
       client.jwtAuth = "";
+      client.apiKey = newApiKey.id;
+
+      // create streams with a projectId
+      for (let i = 0; i < 5; i += 1) {
+        const document = {
+          id: uuid(),
+          kind: "stream",
+          userId: nonAdminUser.id,
+          projectId: projectId,
+        };
+        await server.store.create(document);
```

Comment:
Would be good to have some of these tests actually create objects through the API, not hardcoding them on the db. To see if `projectId` is added properly to the created objects on that flow as well (is it? 👀)